Documentation/ApiOverview/Logging/Writers/Index.rst

```diff
@@ -167,11 +167,50 @@ Only in the case that all registered writers fail, the log entry plus additional
 will be added to the configured fallback logger (which defaults to
 the :ref:`PhpErrorLog <logging-writers-php>` writer).
 
+.. _logging-writers-usage:
+
+Usage in Custom Class
+---------------------
+
+All log writers can be used in your own classes. You can initialize the loggers in the __construct method of your class :code:`\MyDomain\MyExtension\MyFolder\MyClass`.
+
+.. code-block:: php
+
+    namespace MyDomain\MyExtension\MyFolder;
+
+    use Psr\Log\LoggerAwareTrait;
+
+    class MyClass implements \Psr\Log\LoggerAwareInterface {
+        use LoggerAwareTrait;
+        
+        private $logger;
+
+        ...
```

Comment:
```suggestion
        // ...
```

Reply:
I have comment it out. It shall remain as a comment. Otherwise it is had to understand what is happening here.